app/controllers/manageBusinesses/manage/ConfirmReportingMethodSharedController.scala

```diff
@@ -94,27 +90,37 @@ class ConfirmReportingMethodSharedController @Inject()(val manageIncomeSources:
     Future.successful(
```

Comment:
We should not have to level Future.successful call over async one calls, this would cause issue in the future